### PR TITLE
Fix action to pass if no changes in dependencies

### PR DESCRIPTION
**Pull request by KhoiUna**
```yaml
   - name: Detect changed dependencies
      shell: bash
      run: |
        passed=false        
        for file in ${{ steps.changed-files.outputs.all_changed_files }}; do          
          if [[ $file == *"package.json"* ]] || [[ $file == *"Gemfile"* ]] || [[ $file == *"requirements.txt"* ]]; then
              passed=true
          fi
        done
        echo $passed
        echo "CHECK_PASSED=$passed" >> $GITHUB_ENV
```

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -26,7 +26,24 @@ runs:
...
+    - name: Get changed files
+      id: changed-files
+      uses: tj-actions/changed-files@v35
+
+    - name: Detect changed dependencies
+      shell: bash
+      run: |
+        passed=false        
+        for file in ${{ steps.changed-files.outputs.all_changed_files }}; do          
+          if [[ $file == *"package.json"* ]] || [[ $file == *"Gemfile"* ]] || [[ $file == *"requirements.txt"* ]]; then
```
Comment (ashishbijlani) [inline action.yaml]: makes an assumption on the names of deps file. what if a user has reqs.txt as the dep file. we need to filter based on DEPENDENCY_FILES input.

File: action.yaml [hunk 2]
```diff
@@ -50,35 +67,35 @@ runs:
...
     - name: Install Packj's dependencies
+      if: env.CHECK_PASSED == 'true'
       shell: bash
       run: pip3 install -r ./packj/requirements.txt
 
     - name: Auditing deps with Packj
+      if: env.CHECK_PASSED == 'true'
       shell: bash
       run: |
         input=$(echo "${{ inputs.DEPENDENCY_FILES }}" | sed 's/,/ /g')
```
Comment (ashishbijlani) [inline action.yaml:99]: we are parsing deps file multiple times: first at line 45 name: Parse dependency files then again at line 101. We could store the output in ENV and reuse here.

**Review by ashishbijlani — CHANGES_REQUESTED** [review]

**Comment by KhoiUna** [comment]
I have made some updates!

File: action.yaml [hunk 3]
```diff
@@ -94,6 +87,12 @@ runs:
       shell: bash
       run: pip3 install -r ./packj/requirements.txt
 
+    - name: Move user-defined .packj.yaml to the cloned `packj` folder
+      if: env.CHECK_PASSED == 'true'
+      shell: bash
+      run: |
+        rm ./packj/.packj.yaml
+
```
Comment (ashishbijlani) [inline action.yaml]: I think you were testing something with this code. Let's fix this. Rest looks good.

**Review by ashishbijlani — CHANGES_REQUESTED** [review]

**Review by ashishbijlani — APPROVED** [review]

**Pull request merged**